src/libraries/System.Security.Cryptography.Xml/src/CompatibilitySuppressions.xml

```diff
@@ -1,56 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <Suppression>
-    <DiagnosticId>CP0002</DiagnosticId>
-    <Target>F:System.Security.Cryptography.Xml.SignedXml.XmlDsigSHA256Url</Target>
-    <Left>lib/netstandard2.0/System.Security.Cryptography.Xml.dll</Left>
-    <Right>lib/net462/System.Security.Cryptography.Xml.dll</Right>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0002</DiagnosticId>
-    <Target>F:System.Security.Cryptography.Xml.SignedXml.XmlDsigRSASHA256Url</Target>
-    <Left>lib/netstandard2.0/System.Security.Cryptography.Xml.dll</Left>
-    <Right>lib/net462/System.Security.Cryptography.Xml.dll</Right>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0002</DiagnosticId>
-    <Target>F:System.Security.Cryptography.Xml.SignedXml.XmlDsigSHA384Url</Target>
-    <Left>lib/netstandard2.0/System.Security.Cryptography.Xml.dll</Left>
-    <Right>lib/net462/System.Security.Cryptography.Xml.dll</Right>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0002</DiagnosticId>
-    <Target>F:System.Security.Cryptography.Xml.SignedXml.XmlDsigRSASHA384Url</Target>
-    <Left>lib/netstandard2.0/System.Security.Cryptography.Xml.dll</Left>
-    <Right>lib/net462/System.Security.Cryptography.Xml.dll</Right>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0002</DiagnosticId>
-    <Target>F:System.Security.Cryptography.Xml.SignedXml.XmlDsigSHA512Url</Target>
-    <Left>lib/netstandard2.0/System.Security.Cryptography.Xml.dll</Left>
-    <Right>lib/net462/System.Security.Cryptography.Xml.dll</Right>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0002</DiagnosticId>
-    <Target>F:System.Security.Cryptography.Xml.SignedXml.XmlDsigRSASHA512Url</Target>
-    <Left>lib/netstandard2.0/System.Security.Cryptography.Xml.dll</Left>
-    <Right>lib/net462/System.Security.Cryptography.Xml.dll</Right>
-  </Suppression>
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
-    <Target>T:System.Security.Cryptography.Xml.CryptoSignedXmlRecursionException</Target>
+    <Target>T:System.Security.Cryptography.Xml.CipherData</Target>
     <Left>lib/netstandard2.0/System.Security.Cryptography.Xml.dll</Left>
```

Comment:
These suppressions look like we messed up that package and those APIs shouldn't have been included to the netstandard2.0 ref assembly?

Reply:
In terms of this specific suppression you highlighted, nothing really changed as those three suppressions are just being re-ordered. And in general, yeah looks like the facade is missing a whole bunch of types here se we should check what do we expose in the ref assembly, but I doubt we can make any changes now since we have shipped that nestandard2.0 contract in the past.

Reply:
Yes, I was wondering if we should do something about them as it seems like the netstandard2.0 ref assembly is wrong, no action to take on this PR though.